core/controllers/acl_decorators.py

```diff
@@ -2303,15 +2303,26 @@ def test_can_access(self, story_id, **kwargs):
         Raises:
             PageNotFoundException: The given page cannot be found.
         """
-        story = story_services.get_story_by_id(story_id, strict=False)
+        story = story_fetchers.get_story_by_id(story_id, strict=False)
 
         if story is None:
             raise self.PageNotFoundException
 
-        story_rights = story_services.get_story_rights(
-            story_id, strict=False)
-
-        if story_rights.story_is_published:
+        story_is_published = False
```

Comment:
Make sure to add backend tests.

Reply:
Done

Reply:
Where? I would expect to see them in acl_decorators_test.py.

Reply:
Oh, I added them in story_editor_test. Will add it here as well.

Reply:
Done